adjustments.go

```diff
@@ -93,15 +93,15 @@ func (a *Adjustment) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error
 	*a = Adjustment{
 		AccountCode:            string(v.AccountCode),
 		InvoiceNumber:          int(v.InvoiceNumber),
-		UUID:                   v.UUID,
+		UUID:                   SanitizeUUID(v.UUID),
 		State:                  v.State,
 		Description:            v.Description,
 		AccountingCode:         v.AccountingCode,
 		ProductCode:            v.ProductCode,
 		Origin:                 v.Origin,
 		UnitAmountInCents:      v.UnitAmountInCents,
 		Quantity:               v.Quantity,
-		OriginalAdjustmentUUID: v.OriginalAdjustmentUUID,
+		OriginalAdjustmentUUID: SanitizeUUID(v.OriginalAdjustmentUUID),
```

Comment:
Since Recurly returns UUIDs without dashes, do we need to sanitize when unmarshaling incoming XML?